gen-ai/sample-gemini-multimodal.json

```diff
@@ -0,0 +1,94 @@
+{
+  "variable": [
+    {
+      "value": "https://generativelanguage.googleapis.com/v1beta/models/gemini-pro-vision:generateContent",
```

Comment:
Is it possible to pull out the model name and use as a variable? Might be good enhancement for the next rev (so you can swap with `gemini-1.5-pro` for example


<br class="Apple-interchange-newline">

Reply:
Are you thinking like this? `https://generativelanguage.googleapis.com/v1beta/models/{{gemini-model}}:generateContent`

Reply:
Closing this PR and submitting the collection with this change.